src/app/router.module.ts

```diff
@@ -2,11 +2,11 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 
-import { SignupFormComponent} from './signupform/signupform.component';
+
 
 const routes: Routes = [
     { path: '', component: HomeComponent },
-    { path: '', component: SignupFormComponent }
```

Comment:
You'll still need this in the router, just with a different path, like 'register' instead of ''